.gitignore

```diff
@@ -1,2 +1,5 @@
+.idea
+.DS_Store
 target
-idea
\ No newline at end of file
+.env
+*.iml
```

Comment:
Seems like your code editor doesn't add new lines at the end of the file. It's a minor, though.